### PR TITLE
GeckoLocalization

### PR DESCRIPTION
**Pull request by stasm**
Changes required to make the demo for MozLondon work.

### CODE REVIEW

**Comment by stasm** [comment]
@zbraniecki Do you think we could merge this now?

File: src/runtime/gecko/localization.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { Localization, contexts, translateDocument }
+  from '../../bindings/html';
+import { valueFromContext } from '../../lib/format';
+
+// Services.jsm is imported in entry points: l20n-html and l20n-xul
```
Comment (zbraniecki) [inline src/runtime/gecko/localization.js]: Why not load it here? rollup gets confused about undefined variable `Services` and can reassign it's name...

**Comment by zbraniecki** [comment]
r+

File: src/runtime/gecko/localization.js (hunk 1 above)
Comment (stasm) [inline src/runtime/gecko/localization.js]: We need `Services` for `Service.obs.addObserver` in the entry points.  Can I just import it twice?  It's memoized anyways, right?

**Pull request merged**